src/tests/test_taskrunner.py

```diff
@@ -0,0 +1,123 @@
+import sys
+from . import base
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
+import taskrunner as tr
+
+SUCCESS_RC = 0
+
+def capture_stdout(f):
+    strbuffer = StringIO()
+    sys.stdout = strbuffer
+    try:
+        return {"result": f(),
+                "stdout": strbuffer.getvalue()}
+    except BaseException as e:
+        return {"result": e,
+                "stdout": strbuffer.getvalue()}
+    finally:
+        sys.stdout = sys.__stdout__
+
+class TaskRunner(base.BaseCase):
+    def setUp(self):
+        pass
+
+    def tearDown(self):
+        pass
+
+    def test_list_tasks(self):
+        "printing the list of commands can be invoked in many different ways"
+        list_tasks_invocations = [
+            "", # no args
+            "-l", "--list",
+            "-h", "--help", "-?"]
```

Comment:
`-h` does not list the tasks currently:
```
$ ./bldr -h
Usage: fab [options] <command>[:arg1,arg2=val2,host=foo,hosts='h1;h2',...] ...

Options:
  -h, --help            show this help message and exit
  -d NAME, --display=NAME
...
```
I'm not aware of us using this option in any script.

Reply:
> -h does not list the tasks currently:

neither does `--help`, I'll remove them both